package-lock.json

```diff
@@ -12,7 +12,7 @@
         "@metamask/eth-sig-util": "^5.0.2",
         "@metamask/jazzicon": "^2.0.0",
         "@primer/css": "^20.8.0",
-        "@taikai/dappkit": "^2.1.10",
```

Comment:
make sure package-lock is correct because it's weird that we're _removing dappkit_